starport-core/src/main/scala/com/krux/starport/UpdateStatusPipeline.scala

```diff
@@ -0,0 +1,141 @@
+package com.krux.starport
+
+import com.codahale.metrics.MetricRegistry
+import com.krux.starport.db.record.ScheduledPipeline
+import com.krux.starport.db.table.{Pipelines, ScheduledPipelines}
+import com.krux.starport.metric.{ConstantValueGauge, MetricSettings}
+import com.krux.starport.util.{AwsDataPipeline, ErrorHandler, PipelineState}
+import slick.jdbc.PostgresProfile.api._
+import com.krux.starport.util.PipelineState.State
+
+import java.time.LocalDateTime
+
+object UpdateStatusPipeline extends StarportActivity {
```

Comment:
I don't think you need a separate activity for that, can put those in the cleanup activit

Reply:
Yes, I agree that it can be added in cleanup.
But added it as a separate activity as cleanup is a stable activity running for years now, just do not wanted to touch it or break it. 
Also keeping it separate so that if someone do not want the update status feature they can easily remove the activity 'UpdateStatusPipeline' from Starport.scala, this may save us some AWS API call as well.
I can plan it once this new activity runs smooth for few months.
@realstraw  Kindly suggest.

Reply:
I don't think we want to add one additional activity. 3 activity that needs to finish within one hour seems a lot and and will increase the number of API calls we are sending to AWS with the risk of throttling.

Reply:
Ok, let me try attaching new one inside cleanup.